src/core/client/stream/components/Comment.tsx

```diff
@@ -24,6 +25,7 @@ const Comment: StatelessComponent<CommentProps> = props => {
       <Typography className={styles.author} gutterBottom>
         {props.author && props.author.username}
       </Typography>
+      <Timestamp date={props.createdAt} />
```

Comment:
We need to figure out a way to have `responsiveness` here.

Reply:
I'm having a reusable component in mind, that we can always reuse.

Reply:
I'll work with you on this, I'm gonna test if one of my components work that I wrote in one of my earlier projects.

Reply:
👍